pkgs/by-name/ma/manifold/package.nix

```diff
@@ -41,14 +42,25 @@ stdenv.mkDerivation (finalAttrs: {
     test/manifold_test --gtest_filter=-CrossSection.RoundOffset
   '';
 
+  passthru = {
+    tbb = tbb_2021;
+    tests = {
```

Comment:
is this passthru used anywhere?

Reply:
see pkgs/top-level/python-packages.nix

Reply:
the `tests` passthru is by default built by ofborg, meaning a bump to `manifold` would also build `manifold3d` which is nice to have in case no one thinks of running nixpkgs-review